### PR TITLE
[pcapplusplus] Update to 22.05

### PR DESCRIPTION
**Pull request by seladb**
Update PcapPlusPlus port to v22.05

- #### What does your PR fix?
  Update PcapPlusPlus to v22/05

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  <all / linux, windows, ...>, <Yes/No>

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by seladb** [comment]
PcapPlusPlus versions do not follow SemVer, they follow CalVer. The build is failing because of that. What's the way to fix it?

File: ports/pcapplusplus/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "pcapplusplus",
-  "version": "21.11",
-  "port-version": 1,
+  "version": "22.05",
+  "port-version": 0,
```
Comment (Thomas1664) [inline ports/pcapplusplus/vcpkg.json:4]:
```suggestion
  "version": "22.05",
```

**Comment by JonLiu1993** [comment]
> PcapPlusPlus versions do not follow SemVer, they follow CalVer. The build is failing because of that. What's the way to fix it?

"22.05" does‘t conform to the version format rules and cannot use ./vcpkg format-manifest ports/pcapplusplus/vcpkg.json to format its format, so I changed it to "version-string" and then re-submitted push after formatting vcpkg.json , you can refer to the version format rules [here](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md)

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**